tests/validation_v2/utils/gce-10acre-ranch/gce-10acre-ranch

```diff
@@ -0,0 +1,387 @@
+#!/bin/bash
```

Comment:
@cjellick: ok, I will remove this, add in bootstrap, test deploy and update in my next commit.